_data/navigation.yml

```diff
@@ -27,8 +27,8 @@ primary:
       - text: Getting Started at TTS
         url: getting-started/
         internal: true
-      - text: Onboarding class schedule
-        url: onboarding-schedule/
+      - text: TTS Classes for TTS Onboarding
```

Comment:
```suggestion
      - text: TTS onboarding classes
```
Felt a bit redundant with the two TTS's in there, and should be sentence case

Reply:
Good thought. Maybe an alternate could be TTS Classes for Onboarding, as they're primarily for onboarding bu not just "onboarding classes" and really for anyone to join and learn more about the organization.

Reply:
How about just "TTS Classes"?